RUFAS/routines/animal/feed_emissions_manager.py

```diff
@@ -0,0 +1,137 @@
+import requests
+
+from RUFAS.input_manager import InputManager
+from RUFAS.output_manager import OutputManager
+
+im = InputManager()
+om = OutputManager()
+
+
+class FeedEmissionsManager:

```

Comment:
This class seems to _estimate_ or _calculate_ the emissions that are caused by the purchased feed, it doesn't _manage_ it.
```suggestion
class PurchasedFeedEmissionsEstimator:

```